docker/base-image/admin-tools.Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Alpine base image
+FROM alpine:3.12 AS base-admin-tools
+
+RUN apk add --update --no-cache \
+    ca-certificates \
+    tzdata \
+    bash \
+    curl \
+    vim
+
+# set up nsswitch.conf for Go's "netgo" implementation
+# https://github.com/gliderlabs/docker-alpine/issues/367#issuecomment-424546457
+RUN test ! -e /etc/nsswitch.conf && echo 'hosts: files dns' > /etc/nsswitch.conf
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apk add --update --no-cache \
```

Comment:
i think all the things installed here should be moved to the first `RUN apk add` line